pkg/sqlite/migrationsPostgres/1_initial.up.sql

```diff
@@ -0,0 +1,475 @@
+  created_at timestamp not null,
+  updated_at timestamp not null, code text, photographer text,
+  foreign key(studio_id) references studios(id) on delete SET NULL,
+  foreign key(folder_id) references folders(id) on delete SET NULL
+);
+CREATE TABLE gallery_urls (
+  gallery_id integer NOT NULL,
+  position integer NOT NULL,
+  url varchar(255) NOT NULL,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE,
+  PRIMARY KEY(gallery_id, position, url)
+);
+CREATE TABLE IF NOT EXISTS scenes (
+  id serial not null primary key,
+  title varchar(255),
+  details text,
+  date date,
+  rating smallint,
+  studio_id integer,
+  organized boolean not null default FALSE,
+  created_at timestamp not null,
+  updated_at timestamp not null, 
+  code text, 
+  director text, 
+  resume_time float not null default 0, 
+  play_duration float not null default 0, 
+  cover_blob varchar(255) REFERENCES blobs(checksum),
+  foreign key(studio_id) references studios(id) on delete SET NULL
+);
+CREATE TABLE IF NOT EXISTS groups (
+  id serial not null primary key,
+  name varchar(255) not null,
+  aliases varchar(255),
+  duration integer,
+  date date,
+  rating smallint,
+  studio_id integer REFERENCES studios(id) ON DELETE SET NULL,
+  director varchar(255),
+  "description" text,
+  created_at timestamp not null,
+  updated_at timestamp not null, 
+  front_image_blob varchar(255) REFERENCES blobs(checksum), 
+  back_image_blob varchar(255) REFERENCES blobs(checksum)
+);
+CREATE TABLE IF NOT EXISTS group_urls (
+  "group_id" integer NOT NULL,
+  position integer NOT NULL,
+  url varchar(255) NOT NULL,
+  foreign key("group_id") references "groups"(id) on delete CASCADE,
+  PRIMARY KEY("group_id", position, url)
+);
+CREATE TABLE IF NOT EXISTS groups_tags (
+  "group_id" integer NOT NULL,
+  tag_id integer NOT NULL,
+  foreign key("group_id") references "groups"(id) on delete CASCADE,
+  foreign key(tag_id) references tags(id) on delete CASCADE,
+  PRIMARY KEY("group_id", tag_id)
+);
+CREATE TABLE performer_urls (
+  performer_id integer NOT NULL,
+  position integer NOT NULL,
+  url varchar(255) NOT NULL,
+  foreign key(performer_id) references performers(id) on delete CASCADE,
+  PRIMARY KEY(performer_id, position, url)
+);
+CREATE TABLE studios_tags (
+  studio_id integer NOT NULL,
+  tag_id integer NOT NULL,
+  foreign key(studio_id) references studios(id) on delete CASCADE,
+  foreign key(tag_id) references tags(id) on delete CASCADE,
+  PRIMARY KEY(studio_id, tag_id)
+);
+CREATE TABLE IF NOT EXISTS scenes_view_dates (
+  scene_id integer not null,
+  view_date timestamp not null,
+  foreign key(scene_id) references scenes(id) on delete CASCADE
+);
+CREATE TABLE IF NOT EXISTS scenes_o_dates (
+  scene_id integer not null,
+  o_date timestamp not null,
+  foreign key(scene_id) references scenes(id) on delete CASCADE
+);
+CREATE TABLE performer_stash_ids (
+  performer_id integer,
+  endpoint varchar(255),
+  stash_id varchar(36),
```

Comment:
Recommend using the `uuid` column type to store UUIDs, rather than strings

Reply:
I dont want to divert from the original schema too much.

Reply:
It's mostly an optimization thing, since PG has a native uuid type while sqlite doesn't. Storing UUIDs in a varchar or text takes ~37 bytes (36 chars + 1 byte for size), while using uuid takes only 16 bytes. Not a biggie